packages/common/types/account.ts

```diff
@@ -164,6 +164,7 @@ export type Customer = {
   metadata: Record<string, unknown>;
   lastName?: string;
   fullName?: string;
+  isPremium?: boolean;
```

Comment:
This seems to be risky in terms of getting future updates from an upstream repository.

At the same time I see `metadata` field above, what seems to me like a flexible property to add custom values. What can you tell me about it?

Reply:
the interface supports any key as typed as [x:string], I have put this to get type suggestion, can be removed too